### PR TITLE
feat: Allow passing a window argument to the moveoutofgroup dispatcher

### PR DESCRIPTION
**Pull request by morr0ne**
Very simple change that allows for moving a specific window out of a group. It follows the same convention as the `pin` and `togglefloating` dispatchers by accepting either no args or "active" to act on the focused windows and still remain backwards compatible

### CODE REVIEW

File: src/managers/KeybindManager.cpp [hunk 1]
```diff
@@ -2177,7 +2177,12 @@ void CKeybindManager::moveOutOfGroup(std::string args) {
     if (!*PIGNOREGROUPLOCK && g_pKeybindManager->m_bGroupsLocked)
         return;
 
-    const auto PWINDOW = g_pCompositor->m_pLastWindow;
+    CWindow* PWINDOW = nullptr;
+
+    if (args != "" && args != "active" && args.length() > 1)
```
Comment (vaxerski) [inline src/managers/KeybindManager.cpp:2182]: args.empty() not != "", but you already check for .length() so why is it there even

Comment (morr0ne) [reply src/managers/KeybindManager.cpp:2182]: That's a good question. I wondered the same. The other two implementation did the same thing so I figured there must have been a good reason and just left it there

Comment (morr0ne) [reply src/managers/KeybindManager.cpp]: Fixed it to use the empty and also did the same with the other implementations

**Comment by vaxerski** [comment]
honestly I'd rather check for `length() > 1`, some of these have had instances of single-char args before

**Comment by morr0ne** [comment]
Should be good to go

**Review by vaxerski — APPROVED** [review]
lgtm, needs a wiki mr

**Comment by morr0ne** [comment]
Done: https://github.com/hyprwm/hyprland-wiki/pull/475

**Pull request merged**